src/mediapipe_internal/mediapipegraphconfig.cpp

```diff
@@ -30,4 +46,75 @@ void MediapipeGraphConfig::setSubconfigPath(const std::string& subconfigPath) {
     FileSystem::setPath(this->subconfigPath, subconfigPath, this->basePath);
 }
 
+bool MediapipeGraphConfig::isReloadRequired(const MediapipeGraphConfig& rhs) const {
+    // Checking OVMS configuration part
+    if (this->graphName != rhs.graphName) {
+        SPDLOG_DEBUG("MediapipeGraphConfig {} reload required due to name mismatch", this->graphName);
+        return true;
+    }
+    if (this->basePath != rhs.basePath) {
+        SPDLOG_DEBUG("MediapipeGraphConfig {} reload required due to basePath mismatch", this->graphName);
+        return true;
+    }
+    if (this->graphPath != rhs.graphPath) {
+        SPDLOG_DEBUG("MediapipeGraphConfig {} reload required due to graphPath mismatch", this->graphName);
+        return true;
+    }
+    if (this->subconfigPath != rhs.subconfigPath) {
```

Comment:
Should we reload if somebody changes empty subconfigPath to `./subconfig.json`? In fact, both are equivalent (no subconfig path means it will be ./subconfig.json)